pkg/rules/slp089.go

```
@@ -0,0 +1,381 @@
+func hasLineAddEquivalent(h diff.Hunk, idx int) bool {
+	for j := idx + 1; j < len(h.Lines); j++ {
+		ln := h.Lines[j]
+		if ln.Kind == diff.LineAdd && ln.Content == h.Lines[idx].Content {
+			return true
+		}
+	}
+	return false
+}
+
+// shouldReportExport determines if an export Line should be reported based on its Kind
+func shouldReportExport(line diff.Line, hasLineAddEq func(int) bool) bool {
```

golangci-lint (2.11.4)

[error] func shouldReportExport is unused

(unused)